### PR TITLE
Extend payments call - PW6909

### PR DESCRIPTION
**Pull request by vkuncham**
## Tested scenarios

**Fixed issue**:

### CODE REVIEW

File: adyen-appexchange/main/default/classes/AdyenPaymentUtility.cls [hunk 1]
```diff
@@ -110,8 +100,11 @@ public with sharing class AdyenPaymentUtility {
     *  @return Adyen_Adapter__mdt for the passed metadata type with all fields.
     */
     public static Adyen_Adapter__mdt retrieveGatewayMetadata(String metaType) {
-
+        String merchantAccountName = TEST_MERCHANT_ACCOUNT;
```
Comment (ShubhamVijaivargiya) [inline adyen-appexchange/main/default/classes/AdyenPaymentUtility.cls:103]: Single quotes are missing?

**Review by ShubhamVijaivargiya — CHANGES_REQUESTED** [review]
Please address review comments.

**Review by ShubhamVijaivargiya — APPROVED** [review]

**Review by maassenbas — APPROVED** [review]

File: adyen-appexchange/main/default/classes/AdyenPaymentUtility.cls [hunk 2]
```diff
@@ -110,8 +100,11 @@ public with sharing class AdyenPaymentUtility {
...
         if(Test.isRunningTest()) {
+            if (skipMerchantAccount) {
```
[the remark this answers is not in the document]
Comment (vkuncham) [reply adyen-appexchange/main/default/classes/AdyenPaymentUtility.cls:105]: To test for a failed capture request, we want a blank value in the merchantAccount field, however this is part of the custom metadata that is queried in this method, we cannot update custom metadata through code, hence leveraging this boolean variable to set the merchantAccount name based on the scenario we are testing.

File: adyen-appexchange/main/default/classes/AdyenPaymentUtility.cls (hunk 1 above)
Comment (vkuncham) [reply adyen-appexchange/main/default/classes/AdyenPaymentUtility.cls]: Fixed.

**Pull request merged**